build/commands/lib/config.js

```diff
@@ -177,6 +176,35 @@ Config.prototype.isOfficialBuild = function () {
   return this.buildConfig === 'Release'
 }
 
+Config.prototype.isStableChannel = function () {
```

Comment:
why `Stable` instead of `Release`? I think it's better if we use consistent terminology

Reply:
Removed this functions.